src/plugins/initiatives/initiative.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import {type NodeAddressT} from "../../core/graph";
+
+/**
+ * This makes the assumption a Champion cannot fail in championing.
+ * Instead of a success status, they should be removed if unsuccessful.
+ *
+ * There is also no timestamp for completion or each edge.
+ * They will all use the initiative creation date instead.
+ * We can support accurate edge timestamps by interpreting wiki histories.
+ * However the additional complexity and requirements put on the tracker
+ * don't seem worthwhile right now.
+ * Especially because cred can flow even before bounties are released.
+ * See https://discourse.sourcecred.io/t/write-the-initiatives-plugin/269/6
+ * This will likely create dangling edges, as the initiative may be created
```

Comment:
Because of the timestamp of edges.
For example: https://discourse.sourcecred.io/t/write-the-initiatives-plugin/269 was created Oct 6th. It now has a dependency on https://discourse.sourcecred.io/t/unified-reference-detection/301 created Oct 19th.

Oct 6th - Oct 13th is an interval where the Initiatives wiki exists, but the destination unified references wiki does not.

We could add a requirement (considering the below just-in-time detection) where only edges for nodes that exist can be returned. If we want to eliminate the dangling edges.
Or we can try to parse the history of wiki edits to find out when the link to unified references was added (which I argued is probably not worth the complexity).

Reply:
Got it. Technically, whether an edge is dangling is just about whether the endpoints exist in the graph, not what their timestamps are. So if we have an edge with timestamp `t0` connecting nodes with timestamps `t1`, the edge is not considered dangling, since the nodes do exist in the graph.

One thing we could do for now is give the edge a timestamp which is the later of the initiative creation time or the reference's creation time.

Reply:
Ah I see. Well then only 404 links would potentially create dangling edges.
Both approaches of checking for existence and taking the later of the two creation dates depend on the capabilities of reference detection. So let's get back to that in unified reference detection post.